assets/js/phoenix_live_view.js

```diff
@@ -507,6 +507,9 @@ export class LiveSocket {
         pageY: e.pageY,
         screenX: e.screenX,
         screenY: e.screenY,
+        offsetX: e.offsetX,
+        offsetY: e.offsetY,
+
```

Comment:
✂️ extra space pleaze!

Reply:
For future readers - these are CSS pixels from various points

`client` is from the current viewport
`page` is the encompassing browser window
`screen` is relative to the physical monitor (likely not very useful)
`offset` is relative to a parent container.

Reply:
> ✂️ extra space pleaze!

done :-)